R/zzz.R

```diff
@@ -0,0 +1,5 @@
+httr2_1.0.0 <- NULL
```

Comment:
Possibly a premature optimisation, but `packageVersion()` is reasonable slow, and it seems like `with_mock` gets called in a lot of places.